share/jupyter/nbconvert/templates/base/base.html.j2

```diff
@@ -245,10 +243,9 @@ var element = $('#{{ div_id }}');
 {% set div_id = uuid4() %}
 {% set datatype_list = output.data | filter_data_type %} 
 {% set datatype = datatype_list[0]%} 
-<div id="{{ div_id }}"></div>
-<div class="output_subarea output_widget_view {{ extra_class }}">
+<div id="{{ div_id }}" class="output_subarea output_widget_view {{ extra_class }}">
 <script type="text/javascript">
-var element_id = '#{{ div_id }}';
+var element = $('#{{ div_id }}');
```

Comment:
I was under the impression that jupyter added this automatically as part of the script below. I'll try to confirm that by running this branch locally

Reply:
that would be great if you could test it. But this part was only for widgets, for JS it was still correct.

Reply:
Ah, I sadly have no experience (yet) with widgets